test/testcafe/spec/IdentifyRegistration_spec.js

```diff
@@ -109,7 +110,7 @@ test.requestHooks(mock)('should show errors if required fields are empty', async
   await t.expect(registrationPage.hasEmailErrorMessage()).eql(true);
 });
 
-test.requestHooks(mock)('should show errors after empty required fields are focused out', async t => {
+test.meta('v3', false).requestHooks(mock)('should show errors after empty required fields are focused out', async t => {
```

Comment:
In v3, Alex made the conscious decision to disable this as it causes unnecessary noise when completing a form.